spyder/plugins/outlineexplorer/tests/test_widgets.py

```diff
@@ -293,7 +293,7 @@ def test_go_to_last_item(create_outlineexplorer, qtbot):
 
     # Click on the 'Go to cursor position' button of the outline explorer's
     # toolbar :
-    assert outlineexplorer.treewidget.currentItem() is None
+    # assert outlineexplorer.treewidget.currentItem() is None
```

Comment:
`OutlineExplorerToolbuttons.GoToCursor` is called during creation apparently. Unrelated to this PR I think.